hack/build.sh.d/kn-plugin-workflow.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+readonly quarkus_version="2.7.6.Final-redhat-00006"
+readonly quarkus_platform_group_id="com.redhat.quarkus.platform"
+export EXTERNAL_LD_FLAGS="${EXTERNAL_LD_FLAGS:-} \
+-X github.com/kiegroup/kie-tools/packages/kn-plugin-workflow/pkg/metadata.QuarkusVersion=${quarkus_version} \
+-X github.com/kiegroup/kie-tools/packages/kn-plugin-workflow/pkg/metadata.QuarkusPlatformGroupId=${quarkus_platform_group_id}"
```

Comment:
@ljmotta I'll be tentatively online next week during KubeCon. And on the PTO afterwards. Please if you can make sure this version are updated for Serverless 1.26 release.